### PR TITLE
refactor: minor code changes

### PR DESCRIPTION
**Pull request by harshalini**
Did some minor changes in code of component library

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
good to merge

File: components/cards/cards.css [hunk 1]
```diff
@@ -5,13 +5,15 @@
     --light-teal: rgba(0, 255, 170, 0.9);
     --light-violet: #CCD1E4;
     --light-grey: #C9CCD5;
+    --blueviolet: blueviolet;
+    --palevioletred: palevioletred;
```
Comment (fahd23) [inline components/cards/cards.css]: If possible use same type of color code

**Review by fahd23 — APPROVED** [review]
GIven one suggestion, Good to merge

File: components/cards/cards.css (hunk 1 above)
Comment (harshalini) [reply components/cards/cards.css]: done, thanks for the suggestion

**Pull request merged**